### PR TITLE
Drop support for Mongoid 3 and 4 + fix after_fork callback

### PR DESCRIPTION
**Pull request by johnnyshields**
- Drop support for Mongoid 3 and 4
- Perform disconnect on after_fork on all Mongoid versions

This gem's code has been quite stable so there's really no reason to continue to support Mongoid 3/4 (Mongoid 4 has been been EOL for 4 years now.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
-### 2.3.2 (Next)
+### 3.0.1 (Next)
 
 * Your contribution here.
 
+### 3.0.0 (2019/02/26)
```
Comment (dblock) [inline CHANGELOG.md:5]: This should be (Next)

File: README.md [hunk 2]
```diff
@@ -12,7 +12,9 @@
 
 ## Requirements
 
-Mongoid 3, 4, 5, 6 or 7.
+Mongoid 5 or later.
+
+(For Mongoid 3 and 4, use 2.x version of this gem)
```
Comment (dblock) [inline README.md:17]: Add a period.

**Review by dblock — CHANGES_REQUESTED** [review]
Can we remove mongoid_compatibility from .gemspec or is it used elsewhere?

**Comment by johnnyshields** [comment]
> Can we remove mongoid_compatibility from .gemspec or is it used elsewhere?

I have removed it, it's not needed anymore.

**Comment by johnnyshields** [comment]
Please merge and cut a release

**Comment by dblock** [comment]
Merged. It's nice to see you here @johnnyshields! I added you to the repo maintainers and to rubygems so you can cut the release. Thanks for your help!

**Comment by johnnyshields** [comment]
Awesome, thanks 👍

**Pull request merged**